libs/sdk-react-native/example/App.js

```diff
@@ -86,10 +87,11 @@ const App = () => {
             let deviceKey = await getSecureItem(GREENLIGHT_DEVICE_KEY_STORE)
             let deviceCert = await getSecureItem(GREENLIGHT_DEVICE_CERT_STORE)
 
-            if (!deviceKey) {
+            if (deviceKey == null) {
                 const greenlightCredentials = await recoverNode(Network.BITCOIN, seed)
 
                 addLine("recoverNode", null)
+                setSecureItem(MNEMONIC_STORE, mnemonic)
```

Comment:
This is already done above if the mnemonic is not defined

Reply:
Thanks, leftover from my experiments to recover. Addressed.